src/constants/routerPaths.js

```diff
@@ -7,7 +7,7 @@ const RouterPaths = {
   tests: '/tests',
   loginPage: '/login',
   registration: '/registration',
-  errorPage: '/error',
```

Comment:
If we caught an error, the app should redirect us to the error page with '/404' endpoint or something

Reply:
@kaspian122 what ya think